README.md

```diff
@@ -32,24 +32,18 @@ Nebula builds with ROS 2 Galactic and Humble.
 > **Note**
 >
 > Boost version 1.74.0 or later is required. A manual install may be required in Ubuntu versions earlier than 22.04.
->
-> **Note**
->
-> A [TCP enabled version of ROS' Transport Driver](https://github.com/mojomex/transport_drivers/tree/mutable-buffer-in-udp-callback) is required to use Nebula.
-> It is installed automatically into your workspace using the below commands. However, if you already have ROS transport driver binaries installed, you will have to uninstall them to avoid conflicts (replace `humble` with your ROS distribution):
-> `sudo apt remove ros-humble-udp-driver ros-humble-io-context`
 
 To build Nebula run the following commands in your workspace:
 
 ```bash
 # In workspace
-mkdir src
-git clone https://github.com/tier4/nebula.git src
+git clone https://github.com/tier4/nebula.git
+cd nebula
 # Import dependencies
-vcs import src < src/build_depends.repos
-rosdep install --from-paths src --ignore-src -y -r
+vcs import < build_depends.repos
+rosdep install --from-paths . --ignore-src -y -r
 # Build Nebula
-colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXPORT_COMPILE_COMMANDS=1
```

Comment:
`CMAKE_EXPORT_COMPILE_COMMANDS` - I'm guessing this is for Clang? Strangely enough, with this option set I do get a `compile_commands.json` but I also get a CMake Warning: 

> Manually-specified variables were not used by the project:
>    CMAKE_EXPORT_COMPILE_COMMANDS

I added a commit that adds an ignore to the meta project to remove the warning.

Reply:
https://answers.ros.org/question/282295/how-to-setup-intelisense-on-vs-code-for-ros-c/

:innocent:

Reply:
@drwnz This setting applies to all compilers and is used by IDEs/language servers to analyze the code as pointed out by @knzo25 🙌